fourcc.py

```diff
@@ -8,21 +8,24 @@
 fmt_list = []
 mod_list = []
 
-fmt = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_code', flags=re.M)
-mod = re.compile(r'^#define (\w+)\s*(?:\\$\s*)?fourcc_mod_code', flags=re.M)
-mod_broadcom = re.compile(r'^#define (DRM_FORMAT_MOD_BROADCOM\w+(?=\s))', flags=re.M)
+fmt = re.compile(r'^#define (\w+)\s*(?:\\\s*)?fourcc_code')
+mod = re.compile(r'^#define (\w+)\s*(?:\\\s*)?fourcc_mod_code')
+mod_broadcom = re.compile(r'^#define (DRM_FORMAT_MOD_BROADCOM\w+(?=\s))')
 
 with open(fourcc_include) as f:
-	for l in f.readlines():
-		ident = fmt.search(l)
+	str = f.read()
+	for i in range(0, len(str)):
```

Comment:
With this, you can do `fmt.finditer()` or `fmt.findall()`.

Reply:
Done.